trust/service.go

```diff
@@ -8,13 +8,16 @@ import (
 	"github.com/docker/libtrust"
 )
 
+// NotVerifiedError implements the error interface
```

Comment:
This comments doesn't really say anything.
"NotVerifiedError error is returned when …" (I'm still thinking what to put here :sweat_smile: )